packages/mama-core/scripts/re-embed-migration.mjs

```
@@ -0,0 +1,122 @@
+
+  // 1) Decisions -> embeddings(rowid, embedding). Embed async first, write sync after.
+  const decisions = db
+    .prepare('SELECT rowid AS rid, topic, decision, reasoning, outcome, confidence FROM decisions')
+    .all();
+  log(`decisions to re-embed: ${decisions.length}`);
+  const decVecs = [];
+  let i = 0;
+  for (const row of decisions) {
+    const vec = await embedDecision(row);
+    decVecs.push({ rid: row.rid, buf: Buffer.from(vec.buffer, vec.byteOffset, vec.byteLength) });
+    if (++i % 50 === 0) log(`  decisions embedded: ${i}/${decisions.length}`);
```

ESLint

[error] Expected { after 'if' condition.

(curly)